lib/bundler/definition.rb

```diff
@@ -307,6 +307,11 @@ def to_lock
         out << "  #{p}\n"
       end
 
+      if ruby_version
```

Comment:
this will be backwards-incompatible with versions of bundler prior to 1.9.6, I believe